project.rosinstall

```diff
@@ -17,3 +17,8 @@
     local-name: lma_kinematics_plugin
     uri: 'https://github.com/shadow-robot/lma_kinematics_plugin'
     version: F_ik_with_joint_limits
+
```

Comment:
To compile, yes, because the code is there, to run, no, because @nrontsis deleted the [plugin from the URDF](https://github.com/nrontsis/kuka-lwr/commit/bcc23e58724b82152abdc3cf221a6b0cbe715400#diff-67fa2a8f8ccc1a9850322559d37ce38cL7). And also changed the transmissions to [sr types](https://github.com/nrontsis/kuka-lwr/commit/bcc23e58724b82152abdc3cf221a6b0cbe715400#diff-a1e89d4f9cbb8a0d8a0480ab835fa0ddR9). I don't think it is the way to go, if you are not in a hurry I would wait a bit for that.

I will (need to) finish the update on the kuka pkg this weekend, where I plan to discard that dependency as well by adding something similar to what COB people did with the [`enableJointFilter` parameter](https://github.com/ipa320/cob_gazebo_plugins/blob/indigo_dev/cob_gazebo_ros_control/README.md#enable-joint-filtering).